code/game/gamemodes/cult/runes.dm

```diff
@@ -802,7 +802,7 @@ var/list/sacrificed = list()
 			cultist.drop_from_inventory(cultist.handcuffed)
 		if (cultist.legcuffed)
 			cultist.drop_from_inventory(cultist.legcuffed)
-		if (istype(cultist.wear_mask, /obj/item/clothing/mask/muzzle))
+		if (is_muzzle(user.wear_mask))

```

Comment:
You're replacing `cultist` with `user`. This is bad. This line, and the one above it, should be `is_muzzle(cultist.wear_mask)`